sphinxcontrib/drawio/__init__.py

```diff
@@ -250,6 +250,10 @@ def _drawio_export(self, input_abspath, options, out_filename):
         if builder.config._display:
             new_env["DISPLAY"] = ":{}".format(builder.config._display)
 
+        if ('ELECTRON_RUN_AS_NODE' in new_env):
```

Comment:
Please do not use braces in the if-statement and use double quotes. Running `black` locally should pick this up.